src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -25,11 +25,14 @@ internal class EvalVisitor : IRNodeVisitor<ValueTask<FormulaValue>, EvalVisitorC
     {
         public CultureInfo CultureInfo { get; }
 
+        public Features Features { get; }
```

Comment:
>Features [](http://example.com/codeflow?start=15&length=8)

Just curious if we use this yet in this class? 
I think all of the features can be handled by the IR / front-end , and don't need eval changes yet.

Reply:
We need it in public static Func<EvalVisitor, EvalVisitorContext, IRContext, TableValue[], ValueTask<FormulaValue>> StandardSingleColumnTable<T>(Func<EvalVisitor, EvalVisitorContext, IRContext, T[], FormulaValue> targetFunction) when calling var columnNameStr = BuiltinFunction.GetOneColumnTableResultName(runner.Features.HasFlag(Features.ConsistentOneColumnTableResult));

Reply:
Now that StandardSingleColumnTable doesn't need this, 
Let's go ahead and remove this. 

Ideally, backend wouldn't need to switch on feature flags (maybe it will someday) because then every backend has to worry about it. So if we can handle in the front end, that's ideal.

Reply:
Removed